homeassistant/components/evohome/water_heater.py

```diff
@@ -0,0 +1,107 @@
+"""Support for WaterHeater devices of (EMEA/EU) Honeywell evohome systems."""
+from datetime import datetime, timedelta
+import logging
+
+import requests.exceptions
+
+from homeassistant.components.water_heater import (
+    SUPPORT_OPERATION_MODE,
+    WaterHeaterDevice)
+from homeassistant.const import (
+    STATE_OFF, STATE_ON)
+
+from . import (
+    EvoDevice,
+    CONF_LOCATION_IDX)
+from .climate import (
```

Comment:
Platforms shouldn't depend on each other. Move these constants to a common module in the package, eg const.py.

Reply:
Done.